defaults/main.yaml

```diff
@@ -2,8 +2,17 @@
 mariadb:
   prefix:
     config: /usr/local/etc
+  config:
+    filename: my.cnf
```

Comment:
In which case do we need something else here?

Reply:
MariaDB 10.4 has got a different file and directory layout

Reply:
Yes, this file is included in the my.cnf in MariaDB 10.4 and if we put those options to my.cnf the options would exist multiple times